newIDE/app/src/AssetStore/index.js

```diff
@@ -87,41 +120,77 @@ export const AssetStore = ({
             <Background
               noFullHeight
               noExpand
-              width={windowWidth === 'small' ? 150 : 250}
+              width={!isFiltersOpen ? 50 : windowWidth === 'small' ? 150 : 250}
             >
-              <ScrollView>
-                <Subheader>
-                  <Trans>Categories</Trans>
-                </Subheader>
-                <CategoryChooser
-                  allItemsLabel={<Trans>All assets</Trans>}
-                  allFilters={filters}
-                  filtersState={filtersState}
-                  error={error}
-                />
-                <Subheader>
-                  <Trans>Filters</Trans>
-                </Subheader>
-                <FiltersChooser
-                  allFilters={filters}
-                  filtersState={filtersState}
-                  error={error}
-                />
-              </ScrollView>
-            </Background>
-            <BoxSearchResults
-              baseSize={128}
-              onRetry={fetchAssetsAndFilters}
-              error={error}
-              searchItems={searchResults}
-              renderSearchItem={(assetShortHeader, size) => (
-                <AssetCard
-                  size={size}
-                  onOpenDetails={() => onOpenDetails(assetShortHeader)}
-                  assetShortHeader={assetShortHeader}
-                />
+              {!isFiltersOpen ? (
+                <Line justifyContent="center">
+                  <IconButton onClick={() => setIsFiltersOpen(true)}>
+                    <Tune />
+                  </IconButton>
+                </Line>
+              ) : (
+                <ScrollView>
+                  <Line justifyContent="space-between" alignItems="center">
+                    <Column>
+                      <Line alignItems="center">
+                        <Tune />
+                        <Subheader>
+                          <Trans>Categories</Trans>
+                        </Subheader>
+                      </Line>
+                    </Column>
+                    <IconButton onClick={() => setIsFiltersOpen(false)}>
+                      <DoubleChevronArrow />
+                    </IconButton>
+                  </Line>
+                  <>
+                    <CategoryChooser
+                      allItemsLabel={<Trans>All assets</Trans>}
+                      allFilters={filters}
+                      filtersState={filtersState}
+                      setIsOnHomePage={setIsOnHomePage}
```

Comment:
setIsOnHomePage seems not a proper name for this prop. Take the point of view of CategoryChooser: 
it does not even know what is a home page. A home page of what?
It just know about its own filters/categories that you give to him. But has no idea when you would want to change if you're on a home page or not - that's not his responsibility! :) 

Instead, what you, as the parent (AssetStore) really want to know is: 
tell me when a tag is clicked and I will react accordingly (here, by... setting we're not on the home page anymore, which is not the business of the child)

Something like:
```suggestion
                      onChoiceChanged={() => setIsOnHomePage(false)}
```